### PR TITLE
Ml for topaz

### PR DESCRIPTION
**Pull request by Halolegend94**
Adds opencv file

### CODE REVIEW

**Comment by Halolegend94** [comment]
This actually requires ffmpeg to be rebuilt to support broadwell.

File: centos7.6/opencv.cyg [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+read -r -d '' MAALI_MODULE_WHATIS << EOF
+OpenCV (Open Source Computer Vision Library) is an open source computer vision and machine learning software library. 
+EOF
+
+# specify which compilers we want to build the tool with
+MAALI_TOOL_CPU_TARGET="$MAALI_DEFAULT_CPU_TARGET"
+MAALI_TOOL_COMPILERS="gcc/7.2.0 gcc/8.3.0"
+MAALI_TOOL_CUDA_COMPILERS="$MAALI_DEFAULT_CUDA_COMPILERS"
+MAALI_TOOL_PREREQ="python/3.6.3 ffmpeg/4.2.3 numpy/1.13.3 cuda/10.2"
```
Comment (dgrimreaper) [inline centos7.6/opencv.cyg:14]: cuda/10.2 shouldn't be here because is in MAALI_TOOL_CUDA_COMPILERS

Comment (Halolegend94) [reply centos7.6/opencv.cyg:14]: Good catch, I forgot it there. Also, I would like the command to fail if `-n` is not specified.

File: centos7.6/opencv.cyg [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+MAALI_TOOL_BUILD_PREREQ="cmake/3.15.0 git/2.12.3"
+# URL to download the source code from
+# Python prior to 3.X was distributed in .tar.xz format.  This is not the case any longer.
+# https://github.com/opencv/opencv/archive/4.3.0.zip
+MAALI_URL="https://github.com/opencv/opencv/archive/$MAALI_TOOL_VERSION.zip"
+
+# location we are downloading the source code to
+MAALI_DST="$MAALI_SRC/$MAALI_TOOL_NAME_ORIG-$MAALI_TOOL_VERSION.zip"
+
```
Comment (dgrimreaper) [inline centos7.6/opencv.cyg:23]: # This will only work with -n, so force it.
MAALI_CUDA_BUILD_ONLY=1

**Pull request merged**